src/mscorlib/shared/System/Diagnostics/Tracing/EventProvider.cs

```diff
@@ -1166,6 +1167,7 @@ internal unsafe protected bool WriteEvent(ref EventDescriptor eventDescriptor, G
             status = m_eventProvider.EventWriteTransferWrapper(
                 m_regHandle,
                 ref eventDescriptor,
+                IntPtr.Zero,
```

Comment:
I confirmed that WriteEventRaw is only used on the TraceLogging code path, so we can ignore this for now.